vnc/vnc_server.c

```diff
@@ -261,6 +261,7 @@ int main(int argc, char **argv)
     sigset_t mask;
     bool verbose;
     fd_set fds;
+    size_t frambuffer_size;
```

Comment:
There's a typo in the variable name (`frambuffer_size`)

Reply:
Thanks! Fixed.